pkg/resource/parameter_group/post_set_output.go

```diff
@@ -15,7 +15,6 @@ package parameter_group
 
 import (
 	"context"
-
```

Comment:
Please keep the imports organized using the Go best practices mentioned in https://github.com/golang/go/wiki/CodeReviewComments#imports